lib/mpl_toolkits/mplot3d/art3d.py

```diff
@@ -163,12 +184,32 @@ class Line3D(lines.Line2D):
 
     def __init__(self, xs, ys, zs, *args, **kwargs):
         """
-        Keyword arguments are passed onto :func:`~matplotlib.lines.Line2D`.
+
+        Parameters
+        ----------
+        xs : array-like
+            The x-data to be plotted.
+        ys : array-like
+            The y-data to be plotted.
+        zs : array-like
+            The z-data to be plotted.
+
+        Additional arguments are passed onto :func:`~matplotlib.lines.Line2D`.
         """
         super().__init__([], [], *args, **kwargs)
         self._verts3d = xs, ys, zs
 
     def set_3d_properties(self, zs=0, zdir='z'):
+        """
+        Set the *z* position and direction of the line.
+
+        Parameters
+        ----------
+        zs : float or array of floats
+            The z-position in 3D space.
```

Comment:
Should this not be in reference to `zdir`, instead of `z` only? (repeat comment below for every `zs`)